Configurations/unix-Makefile.tmpl

```diff
@@ -1231,6 +1270,21 @@ tags TAGS: FORCE
 	-ctags -R .
 	-etags `find . -name '*.[ch]' -o -name '*.pm'`
 
+{- output_off() if $disabled{fips}; "" -}
+fips-checksums: generate_fips_sources
+	if which unifdef > /dev/null; then \
+	    ( cd $(SRCDIR) \
+	      && cat providers/fips.module.sources \
+	             | xargs ./util/fips-checksums.sh \
+	             > providers/fips-sources.checksums \
+	      && openssl sha256 -hex \
```

Comment:
I still don't see the suggested change to `(cd $(SRCDIR/providers); openssl sha256 -hex fips-sources.checksums >fips.checksum); \` here.

Reply:
It's probably even better to use `sha256sum`, since it gives syntactically correct output